tests/utils.py

```diff
@@ -120,7 +121,7 @@ def get_transforms(
 
 def check_all_augs_exists(
     augmentations: typing.List[typing.List],
-    except_augmentations: typing.Optional[typing.Set[typing.Type[albumentations.BasicTransform]]] = None,
+    except_augmentations: Optional[Set] = None,
```

Comment:
I changed type for this function to be more generic, because otherwise `mypy` marked it as wrong and didn't allow to commit.